utils/SwiftBuildSupport.py

```diff
@@ -151,22 +151,22 @@ def get_preset_options(substitutions, preset_file_names, preset_name):
             from swift_build_support.targets import StdlibDeploymentTarget
             for sdk in sdksToConfigure:
                 if sdk == "OSX":
-                    tgts += StdlibDeploymentTarget.OSX.allArchs
+                    tgts += StdlibDeploymentTarget.OSX.targets
```

Comment:
I would prefer to keep these as `allArchs`. The fact that it's a target is already in the name; we specify platform->architecture, and that's the target.

Not only that, but the lists in some cases are things you would never ever build; like `StdlibDeploymentTarget.Linux.allArchs` - nobody is going to build/configure all of those at once, but it's useful when filtering out all linux targets, for example.